.github/labeler.yml

```diff
@@ -0,0 +1,8 @@
+# changes to documentation
+"area/documentation": doc/**/*
+
+# changes to the core lib package
+"area/lib": ./*.go
+
+# changes to the zsh completion
+"area/*sh completion": ./zsh*
```

Comment:
Can we match on the bash completion as well?

Reply:
👍 
just added the matchers for the other autocompletion shells